.travis.yml

```diff
@@ -1,19 +1,13 @@
+dist: xenial
 language: python
-matrix:
-  allow_failures:
-    - python: "3.6"
-    - python: "3.7"
-  include:
-    - python: "2.7"
-    - python: "3.6"
-    - python: "3.7"
-      dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
+python:
+- 2.7
+- 3.5
```

Comment:
shouldn't we keep testing against python 3.6 and 3.7  if there were no failures?

Reply:
@mraarif Since the platform is now running only with python 3.5 we consider that is it not important to test versions 3.6 and 3.7. In a follow-up PR we are going to add support for python 3.8 and remove support for python 2.7.